pages/portfolio.html

```diff
@@ -4,9 +4,55 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="../stylesheets/styles.css" rel="stylesheet" />
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Righteous&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Merriweather&display=swap" rel="stylesheet">
   <title>Document</title>
 </head>
 <body>
-  
+  <div>
+    <header id="header">
+      <h1>Programming Projects</h1>
+    </header>
+    
+    <nav id="navbar">
+      <ul>
+        <li class="nav-link" id="about-link"><b><a href="about.html">About Me</a></b></li>
+        <li class="nav-link" id="index-link"><b><a href="index.html">Index</a></b></li>
+        <li class="nav-link" id="portfolio-link"><b><a href="portfolio.html">Portfolio</a></b></li>
+        </ul>
+    </nav>
+
+      <div id="projects">
+        <section id="section-1">
+          <h2>Individual Projects</h2>
+            <h3><a href="https://github.com/kaitlyngore/task-list-api" target="_blank">Task List API</a></h3>
```

Comment:
These sections look great, but I would tend to approach this using a ul with li tags. If there were more data being shown here, then I could understand having a heading for each project (and details under the project), but these really are lists of content, and so a list of some type feels the most suitable to me.